app/models/pega_table.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class PegaTable < ApplicationRecord
+  # Validate presence of essential fields
+  validates :uuid, presence: true
+  validates :veteranfirstname, presence: true
+  validates :veteranlastname, presence: true
+  validates :response, presence: true
```

Comment:
I forget do we need emails in this table or no?

Reply:
If its needed in the future, we can add it